mythic-melodic/frontend
# Implement features for the playlist

## 1. Merge @ngoclinhphvm latest pull request and resolve conflict

## 2. Implement features for the playlist
Includes: add/edit/delete playlist, add/remove songs in the playlist.
Currently, there is no feature to upload a cover image for the playlist.

**_Testing instruction:_**
- pull the backend "Refactor for Playlists" pull request
- Check each step as in Story 3: Playlist management feature in the file below.
[Melodic Stories.pdf](https://github.com/user-attachments/files/18053585/Melodic.Stories.pdf)
